handbook/company/communications.md

```diff
@@ -524,20 +524,8 @@ US based Fleeties get paid on the last business day of the month. International
 Variable compensation (i.e. commission) is reviewed and submitted for processing by RevOps by the 7th of each month for the previous month or quarter's contributions. US based Fleeties that receive variable compensation will be paid by the 10th. International Fleeties will be paid no later than the last day of the month for the previous month's or quarter’s contributions. 
 
 ### Workiversaries
-We're happy you've ventured a trip around the sun with Fleet- let's celebrate! The  team will post in Slack to highlight your dedication and contribution to Fleet, giving an opportunity for teammates to share their appreciation of your contribution!
-
-Fleet also [evaluates and (if necessary) updates compensation decisions yearly](#compensation-changes), shortly after the anniversary of a team member's start date.
-
-### Compensation changes
-Fleet evaluates and (if necessary) updates compensation decisions yearly, shortly after the anniversary of a team member's start date. The process for that evaluation and update is:
-- On the first Friday of the month, the Head of BizOps posts in the `#help-classified` channel with the list of teammates celebrating anniversaries over the next month.
-- On the day of the fleetiversary, the Head of BizOps or executive team member will post in `#random` celebrating the tenure of the teammate.
-- The Head of BizOps confers with manager or head of department and prepares compensation benchmarking data and schedules time with the CEO and CTO over an existing 1:1 to discuss if an adjustment needs to be made to compensation.
-- During the 1:1 call, founders review benchmarking for role and geography, and decide if there will be an adjustment.
-- Head of BizOps posts to slack in `#help-classified` with the decision on compensation changes and effective date, if any.
-- If a change is to be made, the Head of BizOps communicates decision to the teammate's people manager, who then communicates to their teammate.
-- Head of BizOps updates the respective payroll platform (Gusto or Plane) and [equity spreadsheet](https://docs.google.com/spreadsheets/d/1_GJlqnWWIQBiZFOoyl9YbTr72bg5qdSSp4O3kuKm1Jc/edit?usp=sharing) (internal doc).
-- If an additional equity grant is part of this compensation change, the previous equity and new situation is noted in detail in the "Notes" column of the equity plan, and the "Grant started?" column is set back to "todo" to add it to the queue for the next time grants are processed (quarterly).
+We're happy you've ventured a trip around the sun with Fleet- let's celebrate! The team will post in Slack to highlight your dedication and contribution to Fleet, giving an opportunity for teammates to share their appreciation of your contribution!
+Fleet also [evaluates and (if necessary) updates compensation decisions yearly](https://fleetdm.com/handbook/business-operations#compensation-changes), shortly after the anniversary of a team member's start date.
```

Comment:
instead of linking to the responsibility from this philosophical section, link to this little philosophical section from the bizops responsibility

Reply:
or can remove the link